manage.py

```diff
@@ -1,10 +1,21 @@
 #!/usr/bin/env python3
+"""Django's command-line utility for administrative tasks."""
 import os
 import sys
 
-if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "wwwapp.settings")
-
-    from django.core.management import execute_from_command_line
 
+def main():
+    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'wwwapp.settings_debug')
```

Comment:
If you do this, you'll have to remember to set DJANGO_SETTINGS_MODULE manually every time you do things like `./manage.py migrate` - but I believe you already had to set some environment for the database username and password so I guess it's not a big deal?

Reply:
Yes, I normally either set some environment variables on prod or make few small changes to code.